File: Hafta 6/conditions.py
from cs50 import get_int

x = get_int("x: ")
y = get_int("y: ")

if x < y:
    print("y, x'ten büyüktür.")
elif x > y:
    print("x, y'den büyüktür.")
else:
    print("x, y'ye eşittir")

File: Hafta 1/conditions.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int x = get_int("x: ");
    int y = get_int("y: ");

    if (x < y)
    {
        printf("y, x'ten büyüktür.\n");
    }
    else if (x > y)
    {
        printf("x, y'den büyüktür.\n");
    }
    else
    {
        printf("x, y'ye eşittir.\n");
    }
}